filament/backend/src/metal/MetalEnums.h

```diff
@@ -296,6 +296,27 @@ constexpr inline MTLTextureType getMetalType(SamplerType target) {
     }
 }
 
+inline MTLTextureType getMetalTypeMS(SamplerType target) {
```

Comment:
nit: since `getMetalType()` already a bit vague on call site (well, it's from upstream so don't change that), so maybe a name of `getMetalTypeMultisample()` would be easier to decipher for this version

Reply:
fixed in https://github.com/shapr3d/filament/pull/67/commits/3f472d0b95e6d2f3283658f4591e8952ff216133